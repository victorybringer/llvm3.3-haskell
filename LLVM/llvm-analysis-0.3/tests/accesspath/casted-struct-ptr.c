struct S {
  int x;
  int y;
};

struct T {
  void *s;
  int x;
};

void f(struct T *t) {
  struct S *s = t->s;
  s->y = 10;
}
