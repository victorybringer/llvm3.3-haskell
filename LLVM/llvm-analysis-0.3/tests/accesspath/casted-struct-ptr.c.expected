fromList [ ("@f", ("%struct.T*", [AccessDeref, AccessField 0, AccessDeref, AccessField 1]) ) ]